test/framework/options.py

```diff
@@ -2350,6 +2351,32 @@ def test_new_update_pr(self):
             regex = re.compile(regex, re.M)
             self.assertTrue(regex.search(txt), "Pattern '%s' found in: %s" % (regex.pattern, txt))
 
+    def test_empty_pr(self):
+        """Test use of --new-pr (dry run only) with no changes"""
+        if self.github_token is None:
+            print "Skipping test_new_pr, no GitHub token available?"
+            return
+
+        # get file from develop branch
+        full_url = URL_SEPARATOR.join([GITHUB_RAW, GITHUB_EB_MAIN, GITHUB_EASYCONFIGS_REPO,
+                                       'develop/easybuild/easyconfigs/z/zlib/zlib-1.2.8.eb'])
+        ec = download_file('zlib-1.2.8.eb', full_url, path=os.path.join(self.test_prefix, 'zlib-1.2.8.eb'), forced=True)
```

Comment:
derive `'zlib-1.2.8.eb'` from `full_url`?

```python
ec_fn = os.path.basename(full_url)
ec = download_file(ec_fn, full_url, path=os.path.join(self.test_prefix, ec_fn), forced=True)
```

the `forced` is probably not needed (`self.test_prefix` will be an empty dir)